### PR TITLE
Svh 2877

### PR DESCRIPTION
**Pull request by QimatLuo**

### CODE REVIEW

File: specs/template.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 resources:
     -
-        alias: placeholder
+        alias: provision_time
```
Comment (henryloexo) [inline specs/template.yaml]: provisionTime as we having reportConfig

Comment (QimatLuo) [reply specs/template.yaml]: So we will sync reportConfig to the unique product?

Comment (henryloexo) [reply specs/template.yaml]: I mean camel cast

Comment (henryloexo) [reply specs/template.yaml]: we will change normal product

**Pull request merged**